### DISCUSSION TITLE
Possibilité de dupliquer un graphique vers la page d'accueil et inversement

### DISCUSSION THREAD

**Discussion by AnaelKremer** [Inist-CNRS/lodex#2091] [Ideas]
Il est possible actuellement de dupliquer des graphiques par un simple click que ce soit dans l'onglet accueil ou graphique, mais l'objet dupliqué ne peut pas passer d"une page à l'autre.

Or on rencontre tous des cas où un graphique doit être présent en page graphique et en accueil, mais où l'on doit changer les dimensions d'une page à l'autre ou modifier le titre du graph, ajouter une légende etc. On peut utiliser "copie d'un graphique existant" mais on ne peut modifier les paramètres d'un sans modifier l'autre également. 

Serait-il possible de pouvoir dupliquer un graphique et de le faire glisser vers la page d'accueil pour ne pas avoir à resaisir tous les paramètres ? Ou via un autre moyen ?

**Comment by touv** [comment]
à clarifier dans une carte issue dediée
